### PR TITLE
Breakout Room 1: Update main.py

### PR DESCRIPTION
**Pull request by tgoslee**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
```
Comment (hopeolaide) [inline main.py:8]: I suggest distinguishing that the user enters a 'second' noun, so they don't think it is a repeat or error.

File: main.py [hunk 2]
```diff
@@ -1 +1,23 @@
...
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
```
Comment (hopeolaide) [inline main.py:11]: I suggest distinguishing that the user enters a 'third' noun, so they don't think the prompt is a repeat or error.

File: main.py (hunk 1 above)
Comment (hopeolaide) [reply main.py:8]: Consider creating  a helper function to manage the repeated request for a noun.

File: main.py [hunk 3]
```diff
@@ -1 +1,23 @@
...
+// Displays the story based on the users input
+    print ("------------------------------------------")
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
```
Comment (hopeolaide) [inline main.py:15]: ***Misspelled 'second'

File: main.py [hunk 4]
```diff
@@ -1 +1,23 @@
...
+    print ("Be kind to your",plural_noun,"in",place)
+    print ("Where the weather is always",adjective,".")
+    print ()
+    print ("You may think that is this the",third_noun,",")
```
Comment (hopeolaide) [inline main.py:19]: Incorrect word order: 'this' and 'is' should be switched.

File: main.py [hunk 5]
```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
```
Comment (hopeolaide) [inline main.py:6]: Note that the inputs are not properly validating entries. Check tests and for edge case inputs.

File: main.py [hunk 6]
```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
```
Comment (hopeolaide) [inline main.py:5]: Consider placing all input requests within a single function to create the desired output. A helper function could also be useful for repeated requestions.

File: main.py [hunk 7]
```diff
@@ -1 +1,23 @@
...
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
+// Displays the story based on the users input
+    print ("------------------------------------------")
```
Comment (hopeolaide) [inline main.py:13]: Consider combing the print statements into a single print statement that is a formatted string containing each variable.

File: main.py [hunk 8]
```diff
@@ -1 +1,23 @@
...
+    print ("Be kind to your",noun,"- footed", plural_noun)
+    print ("For a duck may be somebody's", seond_noun,",")
+    print ("Be kind to your",plural_noun,"in",place)
+    print ("Where the weather is always",adjective,".")
+    print ()
+    print ("You may think that is this the",third_noun,",")
+    print ("Well it is.")
+    print ("------------------------------------------")
+// Loop back to "loop = 1"
+    loop = loop + 1
```
Comment (hopeolaide) [inline main.py:23]: Refactor to state: loop += 1

File: main.py (hunk 7 above)
Comment (hopeolaide) [reply main.py:13]: The print statement could be separated into individual lines for each phrase.

**Review by hopeolaide — CHANGES_REQUESTED** [review]
Great work. Now do it again :)
